src/cli/run.js

```diff
@@ -171,3 +177,12 @@ if (validationFailures) {
     // eslint-disable-next-line flowtype-errors/uncovered
     process.exit(1);
 }
+
+if (config.dumpOperations) {
+    const dumpOperations = config.dumpOperations;
+    const parent = dirname(dumpOperations);
+    if (!existsSync(parent)) {
+        mkdir(parent, {recursive: true});
+    }
+    writeFileSync(dumpOperations, JSON.stringify(printedOperations.sort()));
```

Comment:
Would it be possible/advantageous to print the JSON in a more human-readable format, e.g. `JSON.stringify(..., null, 2)`? I find our existing safelists hard to grep/read because there are no line breaks.

Reply:
I'm fine with that